src/main/java/org/currencygoldexchangeapp/handlers/ExchangeRateAPIHandler.java

```diff
@@ -0,0 +1,61 @@
+package org.currencygoldexchangeapp.handlers;
+
+import org.currencygoldexchangeapp.constants.APIConstants;
+import org.currencygoldexchangeapp.constants.LoggerFactory;
+import org.currencygoldexchangeapp.utils.JSONMapper;
+import org.currencygoldexchangeapp.datamodels.CurrencyExchange;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.LocalDate;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ExchangeRateAPIHandler {
+    private static final Logger LOGGER = LoggerFactory.getLogger();
```

Comment:
Result of getLogger is already static, no need to introduce another static

Reply:
I see, its making sense.

In current implementation I made LOGGER static to share it across all instances of ExchangeRateAPIHandler for efficient class-level logging without creating separate logger instances for each object. What do you think about it?